src/repo.rs

```diff
@@ -1020,6 +1020,24 @@ impl Repository {
             Ok(Oid::from_raw(&raw))
         }
     }
+
+    /// Calculate how far ahead and/or behind a `local` commit is compared to `upstream`
+    pub fn graph_ahead_behind(&self, local: Oid, upstream: Oid) -> Result<(u32, u32), Error> {
+        unsafe {
+            let mut ahead: size_t = 0;
+            let mut behind: size_t = 0;
+            try_call!(raw::git_graph_ahead_behind(&mut ahead, &mut behind, self.raw(), local.raw(), upstream.raw()));
+            Ok((ahead as u32, behind as u32)) // who has >4b commits?
```

Comment:
Or hm, actually can this be `uint`? If libgit2 chose to use `size_t`, the corresponding type in Rust is `uint`, so we should probably stick to the same.